VictorovaHomework4_1.cpp

```diff
@@ -0,0 +1,93 @@
+#include <chrono>
+#include <iostream>
+#include <random>
+#include <time.h>
```

Comment:
```suggestion
#include <сtime>
```
это позволяет понять что вы используете С библиотеки